src/parser/shaman/enhancement/modules/Buffs.tsx

```diff
@@ -4,34 +4,48 @@ import CoreBuffs from 'parser/core/modules/Buffs';
 
 class Buffs extends CoreBuffs {
   buffs() {
+    const combatant = this.selectedCombatant;
+
     return [
       {
-        spellId: SPELLS.FLAMETONGUE_BUFF.id,
-        timelineHighlight: true,
+        spellId: SPELLS.ASCENDANCE_TALENT_ENHANCEMENT.id,
+        timelineHightlight: true,
```

Comment:
We fixed this typo in another PR, so you need to change all of these to `timelineHighlight` instead of `timelineHightlight`